pystiche_papers/sanakoyeu_et_al_2018/_data.py

```diff
@@ -84,22 +85,25 @@ def _properties(self) -> Dict[str, Any]:
         return dct
 
 
-def style_image_transform(edge_size: int = 768) -> transforms.ComposedTransform:
-    return transforms.ComposedTransform(
+def style_image_transform(edge_size: int = 768, train: bool = False) -> nn.Sequential:
+    transforms_: List[nn.Module] = [
         ClampSize(),
         transforms.ValidRandomCrop((edge_size, edge_size)),
         OptionalGrayscaleToFakegrayscale(),
-    )
+    ]
+    if train:
+        transforms_.append(augmentation(size=edge_size))
+    return nn.Sequential(*transforms_)
 
 
 def content_image_transform(
-    impl_params: bool = True, edge_size: int = 768
-) -> transforms.ComposedTransform:
-    transform = style_image_transform(edge_size)
+    impl_params: bool = True, edge_size: int = 768, train: bool = False,
+) -> nn.Sequential:
+    transform = style_image_transform(edge_size=edge_size, train=train)
     if not impl_params:
         return transform
 
-    return transforms.ComposedTransform(transforms.Rescale(2.0), transform)
+    return nn.Sequential(transforms.Rescale(2.0), *transform.children())
```

Comment:
During `stylization`, only the smaller side of the image is resized to the `image_size`. See [here](https://github.com/CompVis/adaptive-style-transfer/blob/07a3b3fcb2eeed2bf9a22a9de59c0aea7de44181/model.py#L492-L494).

Reply:
This refers to https://github.com/CompVis/adaptive-style-transfer/blob/07a3b3fcb2eeed2bf9a22a9de59c0aea7de44181/prepare_dataset.py#L133. We should have done this in #177. I'll send a follow-up PR.